README.md

```diff
@@ -42,18 +50,28 @@ y = 10;
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- Replace this comment with your answer -->
+var x = 4; x = 4
+var y = 3; y = 3
+x = y; // x = 3 - changed
+y = 10; // y = 10 changed
+
+values held by the variables change, variables themselves stay the same
 
 ## Question 4
 
 ```js
 var weather;
-weather = "sunny";
-weather === "sunny";
+weather = "sunny";  // the value of this variable is set to the string value "weather"
+weather === "sunny"; // The value is operated apon to hold a strict value of exactly the string "sunny" in the strictest sense.
 ```
 
 What are the values of these expressions?  Explain your answers.
 
 <!-- Replace this comment with your answer -->
+var weather;  //weather is undefined int his line, has no value yet
+weather = "sunny";  //the variable = the string "sunny"
+weather === "sunny"; // the variable is acted upon to equal precisely teh string "sunny" in the strictest sense.
```

Comment:
Careful. `weather` doesn't change in this last line; `===` is just comparing its value to "sunny" and returning either `true` or `false` depending on whether or not it's a match.